### PR TITLE
Removed unused I18N strings from the `loginIncentives.json` files (related to issue #9957)

### PR DESCRIPTION
**Pull request by RaitheOfDureya**
[//]: # (Note: See http://habitica.fandom.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # here, if applicable. This will automatically close the issue if your PR is merged in)
This PR fixes part of issue #9957

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
I have identified the following unused strings in the `loginIncentives.json` file.  No dynamically generated strings were found.

At the moment I only removed the strings from the `locales/en` folder to make it easier to revert if necessary. Once the strings are reviewed, I can delete them from other languages.

- totalCount
- totalCheckins
- totalCheckinsTitle
- checkinReceivedAllRewardsMessage

Besides, `checkinReceivedAllRewardsMessage` appears in two commented blocks (i.e., `profele.vue` and `loginIncentives.js` files). Do you wanna keep it?
```javascript
// if (!currentLoginDay) return this.$t('checkinReceivedAllRewardsMessage');
// change checkinReceivedAllRewardsMessage in website/common/locales/en/loginIncentives.json
```

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID: 6544e17f-237b-4915-ae7e-47744c0dc1bd

### CODE REVIEW

**Comment by paglias** [comment]
`checkinReceivedAllRewardsMessage` -> let's remove it, I don't think we're going to use it in the near future, The other strings look good!

**Comment by RaitheOfDureya** [comment]
@paglias I remove the strings from all the other languages (including the `checkinReceivedAllRewardsMessage`)

File: website/common/locales/en/loginIncentives.json [hunk 1]
```diff
@@ -6,13 +6,10 @@
   "totalCount": "<%= count %> total count",
```
Comment (paglias) [inline website/common/locales/en/loginIncentives.json]: Is this string used anywhere? I can't find it

Comment (RaitheOfDureya) [reply website/common/locales/en/loginIncentives.json]: I think I confused it with occurrences in functions:
e.g.,

 in: `website/client/src/components/userMenu/profileStats.vue`
```javascript
...
  <strong>{{ $t('petsFound') }}:</strong>
                {{ totalCount(user.items.pets) }}
              </div>
...
```
in: `website/client/src/components/userMenu/stats.vue`
```javascript
...
  {{ $t('mountsTamed') }}:
              {{ totalCount(user.items.mounts) }}
...
```

**Comment by RaitheOfDureya** [comment]
now everything should be all right, @paglias

File: website/common/locales/en/loginIncentives.json (hunk 1 above)
Comment (RaitheOfDureya) [reply website/common/locales/en/loginIncentives.json]: I just removed this string from all languages files

**Comment by paglias** [comment]
thanks! Noted

**Pull request merged**